examples/rivets/index.html

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
+<html lang="en" data-framework="rivets">
+	<head>
+		<meta charset="utf-8">
+		<title>Rivets • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<script type="text/template" id="todos">
+			<header id="header">
```

Comment:
* Fixed `Clear completed ()`
* Fixed test, output: https://gist.github.com/altmind/614b15cb5f16d41d4707 It seems that tests/README.md is out of sync. used `mocha allTests` directly